.github/workflows/master.yml

```
@@ -19,7 +19,7 @@ jobs:
 
       - uses: actions/setup-go@v5
         with:
-          go-version: 1.21
+          go-version-file: "go.mod"
```

Any specific reason for having the double quotation mark instead of single one?
```suggestion
          go-version-file: 'go.mod'
```